src/data/detail-page/charts/devOps.ts

```diff
@@ -1,4 +1,61 @@
-import { Chart } from './chartInterfaces';
+import { Chart, ChartDataItem, ChartColor } from './chartInterfaces';
 
-export const devOps: Chart = {
+interface DevOpsChartDataItem extends ChartDataItem {
+  color: ChartColor;
+}
+
+type DevOpsChart = Omit<Chart, 'data'> & { data: DevOpsChartDataItem[]};
```

Comment:
I'm not sure about substituting data type for a given chart. In this case, I'd rather go for optional `color` prop on `ChartDataItem` and handling potential override in the place where we use the data.

Reply:
Done